azure-pipelines.yml

```diff
@@ -65,6 +65,16 @@ jobs:
   - script: bundle exec bin/test/cucumber.rb
     displayName: "exec cucumber"
 
+  # azureStorage* env vars are defined in the pipeline UI as secret variables
+  - bash: "./bin/ci/az-publish.sh"
+    env:
+      AZURE_STORAGE_ACCOUNT: $(azureStorageAccount)
+      AZURE_STORAGE_KEY: $(azureStorageKey)
+      AZURE_STORAGE_CONNECTION_STRING: $(azureStorageConnectionString)
+      SOURCE_BRANCH: $(Build.SourceBranch)
+    displayName: "Publish to Azure Blob Storage"
+    condition: and(succeeded(), eq(variables['Agent.JobName'], variables['mainJob']))
```

Comment:
We need additional condition for the branch. I think artifact publishing should be performed only for `master` / `develop` branch.
We can use the current condition for testing but please add branch condition before merging

Reply:
@jmoody

Reply:
Can we configure this step to only publish when the branch is a tag?

https://github.com/calabash/calabash-ios-server/tags

e.g. `tag/0.21.8` ?

@max-lobanov

Reply:
Added condition for tags:
```
eq(variables['Agent.JobName'], variables['mainJob'])
```